### PR TITLE
feat(sdk): rs-sdk ProtocolVersion::fetch_many() 

### PR DESCRIPTION
**Pull request by lklimek**
## Issue being fixed or feature implemented

Added  ProtocolVersion::fetch_many() to rs-sdk

## What was done?

Added  ProtocolVersion::fetch_many() to rs-sdk

## How Has This Been Tested?

Added unit test

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [x] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/rs-sdk/tests/fetch/protocol_version_votes.rs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+async fn test_protocol_version_votes() {
+    setup_logs();
+
+    let cfg = Config::new();
+    let mut sdk = cfg.setup_api().await;
+
+    let query = ProTxHash::from_slice(&[0u8; 32]).expect("zero protxhash");
+    let votings = ProtocolVersion::fetch_many(&mut sdk, query)
+        .await
+        .expect("fetch protocol version votes by node");
```
Comment (QuantumExplorer) [inline packages/rs-sdk/tests/fetch/protocol_version_votes.rs:17]: I don't really understand here, how do you get the count of the elements you want to fetch, or how do you get from none? I think this interface is very confusing.

**Review by QuantumExplorer — CHANGES_REQUESTED** [review]

File: packages/rs-sdk/tests/fetch/protocol_version_votes.rs (hunk 1 above)
Comment (lklimek) [reply packages/rs-sdk/tests/fetch/protocol_version_votes.rs:17]: You should use `LimitQuery` , I'll add a test

Comment (lklimek) [reply packages/rs-sdk/tests/fetch/protocol_version_votes.rs:17]: You can't get from none, as the .proto requires this field to be filled: 

```
    bytes start_pro_tx_hash = 1;
```

Should we change that?

Comment (lklimek) [reply packages/rs-sdk/tests/fetch/protocol_version_votes.rs]: I added extensive docs, and created a new type `MasternodeProtocolVote`.

Also added conveniance method ` MasternodeProtocolVote::fetch_many_with_limit()`

Hope it's more developer-friendly now.

Comment (QuantumExplorer) [reply packages/rs-sdk/tests/fetch/protocol_version_votes.rs]: bytes start_pro_tx_hash = 1; can be empty, which translates to none for the query.

Comment (lklimek) [reply packages/rs-sdk/tests/fetch/protocol_version_votes.rs]: pushed support for None in protxhash

**Review by QuantumExplorer — APPROVED** [review]

**Pull request merged**